docs/Firebase.md

```diff
@@ -2,17 +2,13 @@
 
 ## Installation
 
-To install the firebase cli and emulator you must have npm installed already and run:
+To install the firebase cli and emulator you must have npm installed alreadynpm run and run:
```

Comment:
A find and replace error?

Reply:
Don't know how that got in there but good spot

Reply:
Corrected